ports.php

```diff
@@ -178,8 +178,7 @@
                  $row = pg_fetch_assoc($result), $i++) {
 
                 /* Port name and Portfile URL */
-                print "<dt><b><a href=\"${trac_url}browser/trunk/dports/" . $row['path'] . "/Portfile\">" . htmlspecialchars($row['name'])
-                . '</a></b> ' . htmlspecialchars($row['version']) . '</dt>';
+                print '<dt><b>' . htmlspecialchars($row['name']) . '</b> ' . htmlspecialchars($row['version']) . "<i>(<a href=\"${github_url_ports}blob/master/" . $row['path'] . "/Portfile\">source</a>)</i></dt>";
```

Comment:
But note that `$trac_url` is used "unprotected" at many places. Not that it's worth putting too much effort into cleaning up this code in case that newer sites are on the horizon.